packages/grpc-js/src/server.ts

```diff
@@ -301,6 +329,9 @@ export class Server {
       this.options['grpc.keepalive_time_ms'] ?? KEEPALIVE_MAX_TIME_MS;
     this.keepaliveTimeoutMs =
       this.options['grpc.keepalive_timeout_ms'] ?? KEEPALIVE_TIMEOUT_MS;
+    this.sessionIdleTimeout =
+      this.options['grpc.max_connection_idle'] ?? MAX_CONNECTION_IDLE_MS;
```

Comment:
The proper name for this option is [`grpc.max_connection_idle_ms`](https://grpc.github.io/grpc/cpp/group__grpc__arg__keys.html#ga8a7a3d98f39302c7859aa09e3f35a12a). Also, please add that option to the lists in `channel-options.ts` and the README.